db/db_test.go

```diff
@@ -227,6 +228,165 @@ func TestPut(t *testing.T) {
 	mustGetVersion(ver3, "test value 2")
 }
 
+func TestCreateVersion(t *testing.T) {
+	secretName := "secret1"
+	checkVersion := func(d *setectest.DB, version api.SecretVersion, want []byte) *api.SecretValue {
```

Comment:
You might want to plumb in the `*testing.T` here too, otherwise it will misattribute the line numbers.